pkgs/development/compilers/graalvm/default.nix

```diff
@@ -0,0 +1,11 @@
+{ lib
+, stdenv
+, callPackage
+, fetchurl
+}:
+
+{
+
+  buildGraalvm = callPackage ./buildGraalvm.nix;
```

Comment:
Not sure why we need to move this. This can be kept under the `graalvmCEPackages`.

Reply:
hum, I thought it was not a good idea because it's not a community-edition graalvm, so this buildGraalvm function can be used for both and should be more generic, WDYT?